docs/source/certs_management.md

```diff
@@ -0,0 +1,381 @@
+
+**Description**: Admin certificate used to administer the ordering service and submit or approve channel updates. The associated admin identity is enrolled with the Organization CA before the node and Organization MSP are created.
+
+**Location**: Dependent on implementation:
+
+<pre>
+ordererca/
+└── ordereradmin
+└── msp
+    ├── admincerts
+    │   └── cert.pem
+    ├── cacerts
+    │   └── localhost-7053.pem
+    ├── keystore
+    │   └── key.pem
+    ├── signcerts
+    │   └── cert.pem
+    └── user
+</pre>
+
+**Impact if expired**: Transactions can continue to work successfully. Cannot modify channels from a client application or manage the orderer from the console.
+
+[Reference](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-org0-s-admin)
+
+
+#### Peer Service Organization Channel Admin Certificate
+
+**Description** - Admin certificate used to administer the peer and install smart contracts on the peer. Can also be configured to manage the application channel that the peer belongs to. The associated admin user is registered with the Organization CA before the peer and Organization MSP are created.
+
+**Location** - Dependent on implementation:
+
+<pre>
+org1ca/
+└── org1admin
+└── msp
+├── admincerts
+│     └── cert.pem
+├── cacerts
+│     └── localhost-7053.pem
+├── keystore
+│     └── key.pem
+├── signcerts
+│     └── cert.pem
+└── user
+</pre>
+
+**Impact if expired**: Transactions can continue to work successfully. Cannot install new smart contracts from a client application or manage the peer from the console.
+
+[Reference](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-org1-s-admin)
+
+
+
+### Client Certificates
+
+**Description**: Three types of Client certificates are issued for each organization:
+
+1. Certificates used to authenticate an identity to the Gateway when submitting transactions
+2. Client-side TLS certificates used to establish a secure network link to a Gateway peer
+3. Certificate for a client SDK user if no gateway is accessed.
+
+Both the client and the CA have to sign a client certificate for it to be valid.
+
+Client certificates expire after one year, using the Hyperledger Fabric CA default settings. Client certificates can be re-enrolled using either command line Hyperledger Fabric CA utilities or the Fabric CA client SDK.
+
+**Impact if expired**: Client certificates must be re-enrolled before expiration or the client application will not be able to connect to the Fabric nodes.
+
+[Reference](https://hyperledger.github.io/fabric-sdk-node/release-2.2/FabricCAClient.html#reenroll__anchor)
+
+
+### Certificate Decoding
+
+X.509 certificates are distinct from registrations, which do not have an expiration. X.509 certificates are created based on the registration and contain information from the CA. This information includes metadata indicating the parent CA and encoded information describing the purpose of the certificate.
```

Comment:
"which do not have an expiration" looks mis-placed. There is no context for expiration discussion here. And it is strange that we only mention registration here. We should also mention enrollment as that is the process that actually creates the certificate based on the registration.

We should mention that the certificate also includes an expiration in the "Not After" field.

Reply:
ok